content/en/monitors/types/process.md

```diff
@@ -19,21 +19,44 @@ further_reading:
 
 ## Overview
 
-Live process monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+Live Process Monitors are based on data collected by the [Process Agent][1]. Create monitors that warn or alert based on the count of any group of processes across hosts or tags.
+
+Live Process Monitors are recommended to:
+
+- Ensure that enough instances of a non-containerized long-lived process are running.
+- Flag when a specific process is running.
+
+**Note**: Only long-lived processes are picked up by the Agent. Monitors on processes that live for less than 20 seconds may be flaky.
 
 ## Monitor creation
 
-To create a [live process monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
+To create a [Live Process Monitor][2] in Datadog, use the main navigation: *Monitors --> New Monitor --> Live Process*.
```

Comment:
Worth mentioning the easier path via Processes view and New Monitor button - this can generate more views for the product and easier to track in RUM?

Reply:
Oh yep! That's a super good point that I didn't think about